src/index.html

```diff
@@ -5,8 +5,368 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page">
+    <header class="header">
+      <a href="#home" class="header_logo">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav_list">
+          <li class="nav_item nav_item--linked">
+            <a href="#apple" class="nav_link nav_link--is-active">Apple</a>
```

Comment:
you need add class is-active
<img width="1432" alt="Screenshot 2023-09-04 at 16 08 15" src="https://github.com/mate-academy/layout_catalog/assets/53228259/72b0552d-eddf-4d7e-bb3b-964b4c7adbab">